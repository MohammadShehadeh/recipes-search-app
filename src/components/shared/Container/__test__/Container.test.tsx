import renderer from 'react-test-renderer';

import { Container } from '..';

describe('Container', () => {
  it('should render "main" to match snapshot', () => {
    const tree = renderer.create(<Container>main</Container>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render "full" to match snapshot', () => {
    const tree = renderer.create(<Container size="full">full</Container>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
