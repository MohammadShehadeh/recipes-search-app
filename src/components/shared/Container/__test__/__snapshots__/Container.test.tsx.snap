// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render "full" to match snapshot 1`] = `
<div
  className="container"
>
  <div
    className="full"
  >
    full
  </div>
</div>
`;

exports[`Container should render "main" to match snapshot 1`] = `
<div
  className="container"
>
  <div
    className="main"
  >
    main
  </div>
</div>
`;
